facp.md

```diff
@@ -0,0 +1,217 @@
+# FACP
+
+The FADT, of which the signature is FACP, refers to the Fixed ACPI Description Table and is pointed by the RSDT.
+FACP can be used for ACPI shutdown etc.
+ACPI shutdown via FACP can be done both in Virtual Machines and Real PCs.
+
+So how can I find the FACP?
+The RSDT has entries that point to ACPI, FADT and more.
+The entries are 4 in number, and each of them are pointing to the tables.
+FACP is one of them and could be found by strncmp like functions.
+Here is the example code:
+```c
+if(strncmp(header->Signature, "FACP", 4)==0){
+	printf("FACP found\n");
+}
+```
+One can find the address of it via comparing strings.
```

Comment:
I'd maybe go a little further, perhaps: "The address of the FADT can be found by comparing the 4 bytes of the signature field with the text "FADT". This is the same as how other SDT headers are found." Maybe worth mentioning that these are *not* null terminated, so you should probably use memcmp rather than stringcmp.